### PR TITLE
Pull Fixture Data from ASpace, remove SVN EADs.

### LINKED ISSUE #278: Update the indexing procedures for the ArchivesSpace EADs
There should be parity between the indexing procedure for the ArchivesSpace EADs and subversion EADs. Further, we should ensure that the fixtures for the SVN EADs are replaced with those exported from ArchivesSpace.

### LINKED ISSUE #288: Update the test fixtures in order to use the ArchivesSpace EADs
This is necessary in order to ensure that there are no test regressions.

### LINKED ISSUE #290: Document the exporting of EADs from an ArchivesSpace installation
As ArchivesSpace is to be a part of the supported finding aid publishing workflow, then I would please contend that there be documentation outlining how one exports EAD XML from an ArchivesSpace resource.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #278, Closes #288, Closes #290

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,87 +40,50 @@ for information regarding the configuration of repositories in ArcLight.
...
-bundle exec cap staging pulfalight:index_pulfa
-```
+A subset of collections (the same that are run in specs) can be indexed into
+  development via `bundle exec rake pulfalight:seed`
+
+#### Adding new EADs to test suite.
+
+1. Open up `app/services/aspace_fixture_generator.rb`
+1. Add EAD ID to the `AspaceFixtureGenerator::EADID` constant.
+1. If you're only interested in a subset of components, add them to the
```
[the remark this answers is not in the document]
Comment (christinach) [reply README.md:84]: I tried adding "C1643" but it timed out. I'm also checking the webpage and https://asqa.lyrtech.org/staff/ it has a 502.

**Review by christinach — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (christinach) [reply README.md:84]: ok the page is back. I reran the task and C1643 is in the fixtures! great work!

File: README.md [hunk 2]
```diff
@@ -40,87 +40,50 @@ for information regarding the configuration of repositories in ArcLight.
...
+   `AspaceFixtureGEnerator::COMPONENT_MAP` constant.
+1. `bundle exec rake pulfalight:fixtures:refresh_aspace_fixtures`
```
Comment (christinach) [inline README.md:86]: a small typoAspaceFixtureGenerator::COMPONENT_MAP

File: app/services/aspace_fixture_generator.rb [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    File.open(fixture_dir.join(fixture_file.repository, "#{fixture_file.eadid}.processed.EAD.xml"), "w") do |f|
+      f.puts(output)
+    end
+  end
+
+  def select_components(fixture_file, components)
+    doc = Nokogiri::XML(fixture_file.content).remove_namespaces!
+    doc.search("//c").each do |container|
+      all_ids = container.search(".//c").map { |x| x["id"] }
+      container.remove if !components.include?(container["id"]) && (all_ids & components).blank?
```
Comment (eliotjordan) [inline app/services/aspace_fixture_generator.rb:91]: thought: There is a lot of logic stuffed into this line! I think it's OK, but it took me a bit to work out. The `and` operator abutting a set intersection especially.

File: lib/tasks/pulfalight.rake [hunk 4]
```diff
@@ -294,8 +290,13 @@ namespace :pulfalight do
     file_paths = Dir.glob(glob_pattern)
 
     file_paths.each do |file_path|
+      # Don't index full versions of seed files if given argument.
+      next if file_path.include?(".processed") && file_path.include?("MC221")
+      # Index all of MC221 - we have several tests for it.
+      next if File.exist?(file_path.gsub(".EAD", ".processed.EAD")) && !file_path.include?("MC221")
```
Comment (eliotjordan) [inline lib/tasks/pulfalight.rake:296]: question: why are certain fixtures considered "processed"? Where does that come from?

File: spec/features/catalog_view_spec.rb [hunk 5]
```diff
@@ -126,15 +127,17 @@
       expect(page.body).to include "Missionaries"
       expect(page.body).to include "Genre Terms"
       expect(page.body).to include "Correspondence"
-      expect(page.body).to include "Names"
-      expect(page.body).to include "Foreign Service Institute"
+      # Commented out these two - names are not currently in our aspace
+      # instance data.
+      # expect(page.body).to include "Names"
+      # expect(page.body).to include "Foreign Service Institute"
```
Comment (eliotjordan) [inline spec/features/catalog_view_spec.rb:133]: suggestion: maybe this could be a `TODO:` so we know to go back and check when names eventually make it into aspace.

**Review by eliotjordan — APPROVED** [review]
That was a substantial PR! Looks good. I just had a few thoughts and questions.

File: README.md [hunk 6]
```diff
@@ -40,87 +40,50 @@ for information regarding the configuration of repositories in ArcLight.
...
-[...] mss
-[...] mudd
-[...] rarebooks
-```
-
-##### Retrieving the Documents from SVN
+#### Configuring ASpace
 
-One must ensure that SVN is installed locally:
+Create a .env file in the root of the project with the following information
```
[the remark this answers is not in the document]
Comment (eliotjordan) [reply README.md:46]: Good idea.

**Review by eliotjordan — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -40,87 +40,51 @@ for information regarding the configuration of repositories in ArcLight.
...
+#### Index "Interesting" EADs
+A sub-section of all our collections have been identified and can be queued up
+  for ingest via:
 
-*In a macOS Environment:*
-```
-brew install lastpass-cli
-```
+  `start redis-server`
+  `bundle exec rake pulfalight:aspace:index_test_eads`
```
Comment (christinach) [inline README.md:58]: @tpendragon can you rephrase `start redis-server` to: 
Start redis: `redis-server` or just `redis-server`
Also can you make sure the two commands appear in separate lines?

**Review by christinach — APPROVED** [review]
@tpendragon I approved the PR. Please check one last comment that I added before merging.

File: README.md (hunk 7 above)
Comment (tpendragon) [reply README.md]: Updated! One more look?

**Review by christinach — APPROVED** [review]
